src/Nethermind/Nethermind.Core/ProductInfo.cs

```diff
@@ -36,7 +36,7 @@ static ProductInfo()
         if (index != -1)
         {
             Commit = Version[(index + 1)..];
-            Version = Version[..Math.Min(index + 9, Version.Length - 1)];
```

Comment:
Pretty sure there is a reason why this is done like this. I just dont know it 😅 @rubo ?

Reply:
Good catch. There is. Needs to be reverted to the original.

Reply:
Why? Original never trims off the commit

Reply:
This line truncates the commit hash to 8 characters if any. With your change, it simply drops the `+abcd1234` part from the version string.

Reply:
Ah ok